docs/samples/server_cert.cnf

```diff
@@ -0,0 +1,21 @@
+[req]
+distinguished_name = req_distinguished_name
+req_extensions = req_ext
+prompt = no
+
+[req_distinguished_name]
+C   = US
+ST  = Durham
+L   = RTP
+O   = ntap
+OU  = Harvest
+CN  = /
+
+[req_ext]
+subjectAltName = @alt_names
+
+[alt_names]
+DNS.1                   = umeng-aff300-05-06-cm.rtp.openenglab.netapp.com
+DNS.2                   = umeng-aff300-05-06
+IP.1                    = 10.193.48.11
```

Comment:
customer would change this i/p based on their network, right ?

Reply:
that's right, the instructions mention that